lib/rbac/filterer.rb

```diff
@@ -120,12 +120,12 @@ class Filterer
     }
 
     # These classes should accept any of the relationship_mixin methods including:
-    #   :parent_ids
-    #   :ancestor_ids
-    #   :child_ids
-    #   :sibling_ids
-    #   :descendant_ids
-    #   ...
+    # * :ancestor_ids   -- children can see these resources
+    #   :child_ids      -- direct parent can see these resources
+    # * :descendant_ids -- all parents can see these resources
+    #   :parent_ids     -- only direct children can see these resources
+    #   :sibling_ids    -- all tenants of the same level can see these
+    # * nil             -- only the same tenant can see the resources
```

Comment:
Great idea.  To build on your idea, perhaps we can make it a table?  Does this help?

```suggestion
    #   Relationship method | Which tenants can see these resources
    #   --------------------------------------------------------------
    # * :ancestor_ids       | child tenants and their descendant tenants
    #   :child_ids          | only the parent tenant
    # * :descendant_ids     | parent tenant and their ancestor tenants
    #   :parent_ids         | only the direct children
    #   :sibling_ids        | all tenants of the same level
    # * nil                 | only the same tenant
```